integration-test/build.gradle.kts

```diff
@@ -263,6 +263,8 @@ tasks.test {
         fileMode = 0b111101101
       }
 
+      jvmArgs(project.property("extraJvmArgs") as List<*>)
```

Comment:
The property `extraJvmArgs` has not been set. Should we remove it?

Reply:
What's the meaning, can you explain more?